api/v1alpha1/managednotification_types_test.go

```diff
@@ -82,6 +83,44 @@ var _ = Describe("OCMAgent Controller", func() {
 			Expect(t.Name).To(Equal(testNotificationName))
 			Expect(err).NotTo(HaveOccurred())
 		})
+		It("will return the correct notification if the resolve body is empty", func() {
+			testManagedNotificationWrb := &v1alpha1.ManagedNotification{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-wrb",
+					Namespace: "test-ns-wrb",
+				},
+				Spec: v1alpha1.ManagedNotificationSpec{
+					Notifications: []v1alpha1.Notification{
+						{
+							Name:       testNotificationNameWrb,
+							Summary:    "Test Summary",
+							ActiveDesc: "Test Firing",
+							Severity:   "Info",
+							ResendWait: 1,
+						},
+					},
+				},
+				Status: v1alpha1.ManagedNotificationStatus{
+					NotificationRecords: []v1alpha1.NotificationRecord{
+						{
+							Name:                testNotificationName,
+							ServiceLogSentCount: 0,
+							Conditions: []v1alpha1.NotificationCondition{
+								{
+									Type:               v1alpha1.ConditionAlertFiring,
+									Status:             corev1.ConditionTrue,
+									LastTransitionTime: &metav1.Time{Time: time.Now()},
+									Reason:             "Test reason",
+								},
+							},
+						},
+					},
+				},
+			}
+			t, err := testManagedNotificationWrb.GetNotificationForName(testNotificationNameWrb)
+			Expect(t.Name).To(Equal(testNotificationNameWrb))
```

Comment:
A very minor thing, but I guess this test could also validate what it expects the contents of t.ResolvedDesc to be in this situation.